src/BlobSagaSpike/SagaManager.cs

```diff
@@ -39,6 +39,7 @@ public Task Process<T>(string correlationId, IMessageHandlerContext context, Fun
         var sagaContainer = await sagaPersister.LoadByCorrelationId(correlationId).ConfigureAwait(false) 
                             ?? new SagaDataContainer(){Id = correlationId};
 
+
         if (!sagaContainer.OutboxState.TryGetValue(messageId, out var outboxState))
```

Comment:
AFAIU current data structures enable atomic operations on the saga data and outbox-state. What this means I think is that OutboxState can play the role of the inbox and we don't need the inbox in that case.